packages/nimble-components/src/menu/styles.ts

```diff
@@ -24,14 +28,18 @@ export const styles = css`
         min-width: 168px;
         width: max-content;
         box-shadow: ${elevation2BoxShadow};
+        position: relative;
```

Comment:
We shouldn't set a position on the `:host`. The user needs to be able to have full control of the element's position configuration. Should only se it on children in our control.

Reply:
Template is pretty minimal lol. Generally not great, but maybe we can style the slot?

Reply:
That's a great point. I updated the code to style the slot + `:host` as the 2 overlapping elements rather than needing to use `:host` and `:host::before`.